src/content/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign.mdx

```diff
@@ -29,15 +29,14 @@ Benefits of enabling automated user management include:
 ## Requirements and recommendations [#requirements]
 
 Requirements and recommendations:
-
 * Requires [Pro or Enterprise edition](https://newrelic.com/pricing).
-* Supports SAML 2.0 standard for single sign on (SSO).
-* Supports SCIM 2.0 standard.
-* [User model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models)-related requirements:
-  * This feature requires you to be on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models) and creates users on that model. If you're on our [original user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models), talk to your New Relic account representative.
-  * Configuration requires the [**Authentication domain manager** role](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#roles) (users in the [default group **Admin**](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#default-groups) have this).
-* There are three identity providers that have a New Relic app: Azure AD, Okta, and OneLogin. For other identity providers, you can use our [SCIM API](/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management).
-* Before enabling this, it helps to first set up [user groups](/docs/accounts/accounts/automated-user-management/roles-permissions-automated-user-management) in your identity provider service and think about which New Relic roles and accounts those groups will have access to. 
+* SCIM: we support the SCIM 2.0 standard. Three identity providers have a New Relic app: Azure AD, Okta, and OneLogin. For other identity providers that use SCIM 2.0, use our [SCIM API](/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management). Exceptions: 
```

Comment:
```suggestion
* SCIM: we support the SCIM 2.0 standard. Three identity providers have a New Relic app: Azure AD, Okta, and OneLogin. 
  * For other identity providers that use SCIM 2.0, use our [SCIM API](/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management). 
  * Exceptions: Ping Identity's PingOne is not supported because it doesn't allow provisioning of groups. 
```

Reply:
Totally a preferential thing for chunking information, so you don't have the lone hollow-circle bullet point.